scripts/find_vae_image_representation_subspaces.py

```diff
@@ -257,7 +258,7 @@ def _process_subspace(
     rng = np.random.default_rng(rng)
 
     evaluation_ixs = rng.choice(len(Z), size=cfg.estimation_N, replace=False)
-    latte_ksg_evaluation_ixs = rng.choice(len(Z), size=2000, replace=False)
+    latte_ksg_evaluation_ixs = rng.choice(len(Z), size=4000, replace=False)
```

Comment:
Question: why is here 4000? In the other file, the default is 8,000, isn't it?

Reply:
Yes, that was a bit of a relic from testing. I will make it a parameter as above.